api/SourceBuffer.json

```diff
@@ -277,7 +277,107 @@
             }
           },
           "status": {
-            "experimental": true,
+            "experimental": false,
+            "standard_track": true,
+            "deprecated": false
+          }
+        }
+      },
+      "changeType": {
+        "__compat": {
+          "mdn_url": "https://developer.mozilla.org/docs/Web/API/SourceBuffer/changeType",
+          "support": {
+            "chrome": [
+              {
+                "version_added": "69",
+                "version_removed": "70",
+                "flags": [
+                  {
+                    "type": "preference",
+                    "name": "MediaSourceExperimental",
+                    "value_to_set": "true"
+                  }
+                ]
+              },
+              {
+                "version_added": "70"
```

Comment:
Ranges are reverse-chronological, please.
See https://github.com/mdn/browser-compat-data/blob/master/schemas/compat-data-schema.md#the-support_statement-object